### DISCUSSION TITLE
selectable bitrate and/or re-encode of the source stream

### DISCUSSION THREAD

**Discussion by cociweb** [music-assistant/support#980] [Q&A] [answered]
Hello,

I would like to start to use this great integration. However, I have a very huge obstacle:
I use squeezelite-esp32 on my devices. Unfortunatelly, the device has some [limitation](https://github.com/sle118/squeezelite-esp32#performances) with the bitrates.
I'm able to re-encode the source stream in LMS and proxify it with the max 96/192 bitrate (depends on the ESP32 16/32bit version) but not with MA.
I mostly use Tunein part from LMS as well, and I'm also able to select the format/bitrate (what are the radios provide) - but in our modern world it is very uncommon to catch a low bitrate stream: re-encode could be a very handy feature.

An additional bonus question: how is it solved with Google (or other) TTS announcements? Are they limitied in 32 kbps by the provider (G.)?

What're your opinions?

**Comment by OzGav** [comment, marked as answer]
Have a look at QUEUE>>SETTINGS>>ADVANCED SETTINGS as this can all be set there

**Reply by cociweb** [reply]
Whoa! Great Thank's in this case I'm not an innovater! :D
Additionally, a sidenote for me: always RTFM!
